### ISSUE TITLE
Autocomplete for the tag command

### ISSUE THREAD

**Issue by Tais993** [Together-Java/TJ-Bot#367]
**Is your feature request related to a problem? Please describe.**
When trying to find a tag, we've to remember the name out of our head.

And you see, me stupid, you stupid, we don't remember the tag

**Describe the solution you'd like**
Implementation of auto-completion which will populate the user's input live when typing.
Example can be found in the gif below:

**Additional context**

*This is an old gif, performance has been improved*

https://user-images.githubusercontent.com/49957334/152675220-d4fb836a-302e-41c5-b704-45f7c39763bb.mp4

# Blocked by #363 and #368 

Until we update JDA and our command system, we won't be able to add this.
Therefore, soon:tm:

**Comment by illuminator3** [comment]
Can confirm, am stupid.

**Comment by Zabuzard** [comment]
Closing in favor of duplicate #594
